sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -86,9 +89,20 @@ public HttpPipeline getHttpPipeline() {
     // Input is String and output is Response<String>.
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<DigitalTwinsResponse<String>> createDigitalTwinWithResponse(String digitalTwinId, String digitalTwin) {
+        return withContext(context -> createDigitalTwinWithResponse(digitalTwinId, digitalTwin, context));
+    }
+
+    // TODO: This is a temporary implementation for sample purposes. This should be spruced up/replaced once this API is actually designed.
+    // Input is Object and output is Response<T>.
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public <T> Mono<DigitalTwinsResponse<T>> createDigitalTwinWithResponse(String digitalTwinId, Object digitalTwin, Class<T> klazz) {
```

Comment:
klazz or clazz>? :)

Reply:
take your pick and let's stick with one :)))

Reply:
`clazz`, then!